src/rgw/driver/posix/rgw_sal_posix.cc

```diff
@@ -422,12 +2034,7 @@ std::unique_ptr<Notification> POSIXDriver::get_notification(
 
 int POSIXDriver::close()
 {
-  if (root_fd < 0) {
-    return 0;
-  }
-
-  ::close(root_fd);
-  root_fd = -1;
+  //root_dir->close();
```

Comment:
what's going on here (commented code)?

Reply:
Left over from when the FDs were in the main SAL classes.  Unused, and removed.